media-sound/ncmpc/ncmpc-0.30.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson ninja-utils
+
+DESCRIPTION="A ncurses client for the Music Player Daemon (MPD)"
+HOMEPAGE="https://www.musicpd.org/clients/ncmpc/ https://github.com/MusicPlayerDaemon/ncmpc"
+SRC_URI="http://www.musicpd.org/download/${PN}/${PV%.*}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="artist-screen chat-screen debug doc +help-screen key-screen lirc lto lyrics-screen mouse nls outputs-screen search-screen song-screen"
+
+RDEPEND=">=dev-libs/glib-2.30:2
+	>=media-libs/libmpdclient-2.9
+	sys-libs/ncurses:0=[unicode]
+	lirc? ( app-misc/lirc )"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS NEWS README.rst doc/config.sample doc/keys.sample )
+
+meson_use() {
+    usex "$1" true false
+}
+
+src_prepare() {
+	eapply ${FILESDIR}/fix_gcc_warning.patch
+	eapply ${FILESDIR}/restore_progress_bar_in_colorless_mode.patch
+	eapply_user
+}
+
+src_configure() {
+	local myconf=(
+		-Db_ndebug=$(usex debug false true)
+		-Db_lto=$(meson_use lto)
+		-Dcurses=auto
+		-Dmouse=$(meson_use mouse)
+		-Dcolors=auto
+		-Dmultibyte=$(meson_use nls)
+		-Dlocale=$(meson_use nls)
+		-Dnls=$(meson_use nls)
+		-Dlyrics_screen=$(meson_use lyrics-screen)
+		-Dlirc=$(meson_use lirc)
+		-Dtcp=true
+		-Dasync_connect=true
+		-Dmini=false
+		-Dhelp_screen=$(meson_use help-screen)
+		-Dartist_screen=$(meson_use artist-screen)
+		-Dsearch_screen=$(meson_use search-screen)
+		-Dsong_screen=$(meson_use song-screen)
+		-Dkey_screen=$(meson_use key-screen)
+		-Dlyrics_screen=$(meson_use lyrics-screen)
+		-Doutputs_screen=$(meson_use outputs-screen)
+		-Dchat_screen=$(meson_use chat-screen)
+		-Ddocumentation=$(meson_use doc)
+	)
+	if use lyrics-screen; then
+		myconf+=(
+			-Dlyrics_plugin_dir=/usr/$(get_libdir)/ncmpc/lyrics
+		)
+	fi
+
+	meson_src_configure "${myconf[@]}"
+}
+
+src_install() {
+	if use doc; then
```

Comment:
what's the purpose of calling the standard function by the use flag? Call it unconditionally.